include/zephyr/dt-bindings/interrupt-controller/tcc-tic.h

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2024 Hounjoung Rim <hounjoung@tsnlab.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef TCC_TIC_ENUM_HEADER
+#define TCC_TIC_ENUM_HEADER
+
+/*
+ * DEFINITIONS
+ *
+ */
+
```

Comment:
We should remove these brackets, as they are redundant (to see other related files.), like this
```
#define TIC_INT_TYPE_LEVEL_HIGH 0x1U
```
not
```
#define TIC_INT_TYPE_LEVEL_HIGH (0x1U)
```

e.g.
```
from clock

#define STM32_CLOCK_BUS_IOP     0x034
#define STM32_CLOCK_BUS_AHB1    0x038

from adc

#define DT_ADC_GPIO_PB2      0x03
#define DT_ADC_GPIO_PB3      0x04
#define DT_ADC_GPIO_PB4      0x05
#define DT_ADC_GPIO_PB5      0x06
```